packages/zowe-explorer/__tests__/__unit__/trees/dataset/DatasetActions.unit.test.ts

```diff
@@ -240,7 +240,7 @@ describe("Dataset Actions Unit Tests - Function createMember", () => {
             session: blockMocks.session,
         });
         const nonFavoriteLabel = parent.label;
-        parent.label = `${parent.label}`;
+        parent.label = `${parent.label?.toString()}`;
```

Comment:
Use of string interpolation seems unnecessary here:
```suggestion
        parent.label = parent.label?.toString();
```